src/clr-angular/forms/styles/_containers.clarity.scss

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+.clr-form-control {
+    padding-top: $clr-forms-baseline * 4;
+
+    .clr-control-label {
```

Comment:
same nesting question here. can a few of these styles be moved out? I will just note this once here but this applies basically to all nested styles.